301-storage-spaces-direct-md-zones/azuredeploy.json

```diff
@@ -111,7 +111,7 @@
       "metadata": {
         "description": "Location of resources that the script is dependent on such as linked templates and DSC modules"
       },
-      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/301-storage-spaces-direct-md-zones"
+      "defaultValue": "https://raw.githubusercontent.com/fepere/azure-quickstart-templates/master/301-storage-spaces-direct-md-zones"
```

Comment:
revert this change

Reply:
Hello,
I revert back, I was made that due the previous was not working, so I can moved forward. Thx